tests/test_goapidiff.py

```diff
@@ -0,0 +1,88 @@
+import json
+import os
+
+import pytest
+
+from system.plugins.goapidiff.analyzer import GoApiDiff
+
+
+TESTDATA_DIR = 'testdata'
+
+PATH_PAIRS_IDENTICAL_COMMITS = [
+    [
+        'github.com/coreos/etcd/b4bddf685b26b4aa70e939445044bdeac822d042',
+        'github.com/coreos/etcd/b4bddf685b26b4aa70e939445044bdeac822d042',
+    ],
+    [
+        'github.com/google/martian/1afb734cc2eb3a5dab708a901ee07106a94a9186',
+        'github.com/google/martian/1afb734cc2eb3a5dab708a901ee07106a94a9186',
+    ],
+]
+
+PATH_PAIRS_DIFFERENT_PROJECTS = [
```

Comment:
Would be better to provide only (project, commit) pair instead of using predefined path created in prepare function. So it is more reusable.